File: ClientSide/UkrainianStyleRestaurant_2.0_another/app/src/main/java/com/example/ukrainianstylerestaurant/model/CreateDishRequest.java
package com.example.ukrainianstylerestaurant.model;

public class CreateDishRequest {
    public String title;
    public double price;
    public String pepper;
    public String color;
    public int categoryId;
    public String imageBase64;

    public CreateDishRequest(String title, double price, String pepper,
                             String color, int categoryId, String imageBase64) {
        this.title = title;
        this.price = price;
        this.pepper = pepper;
        this.color = color;
        this.categoryId = categoryId;
        this.imageBase64 = imageBase64;
    }
}

File: API/MenuService/DTOs/CreateDishRequest.cs
namespace MenuService.DTOs;

public class CreateDishRequest
{
    public int Id { get; set; }
    public string Title { get; set; }
    public decimal Price { get; set; }
    public string Pepper { get; set; }
    public string Color { get; set; }
    public int CategoryId { get; set; }
    public string? ImageBase64 { get; set; }

    // --- НОВЕ ПОЛЕ ---
    public int StationId { get; set; } // 1=Гарячий, 2=Холодний, 3=Гриль, 4=Піца
}